checkov/github/checks/webhooks_https_orgs.py

```diff
@@ -23,18 +23,20 @@ def __init__(self) -> None:
         )
 
     def scan_entity_conf(self, conf: dict[str, Any], entity_type: str) -> tuple[CheckResult, dict[str, Any]] | None:  # type:ignore[override]
-        if org_webhooks_schema.validate(conf):
-            for item in conf:
-                if isinstance(item, dict):
-                    item_config = item.get("config", {})
-                    if not item_config:
-                        continue
-                    url = item_config.get('url', '')
-                    insecure_ssl = item_config.get('insecure_ssl', '0')
-                    secret = item_config.get('secret', '')
-                    if re.match("^http://", url) or insecure_ssl != '0' and secret != '********':  # nosec
-                        return CheckResult.FAILED, item_config
-            return CheckResult.PASSED, conf
+        ckv_metadata, conf = self.resolve_ckv_metadata_conf(conf=conf)
+        if 'org_webhooks' in ckv_metadata.get('file_name', ''):
+            if org_webhooks_schema.validate(conf):
+                for item in conf:
+                    if isinstance(item, dict):
+                        item_config = item.get("config", {})
+                        if not item_config:
+                            continue
+                        url = item_config.get('url', '')
+                        insecure_ssl = item_config.get('insecure_ssl', '0')
+                        secret = item_config.get('secret', '')
+                        if re.match("^http://", url) or insecure_ssl != '0' and secret != '********':  # nosec
```

Comment:
can you move `"^http://"` to `re.compile()` at the top, so it is scannable by CodeQL